versioned/tiered/src/main/java/com/dremio/nessie/versioned/impl/L1.java

```diff
@@ -152,4 +153,14 @@ int size() {
     return count;
   }
 
+  public <T> L1Builder<T> applyToBuilder(L1Builder<T> builder) {
```

Comment:
I would expect signature to be more like
```
   public <U extends L1Builder> U applyToBuilder(U builder);
```

Reply:
Think, the `<T> L1Builder<T>` variant is correct here, as `L1Builder` has a type-arg.
`<U extends L1Builder>` may cause "use of raw type" compiler warnings.